drivers/storage/ebs/tests/Vagrantfile

```diff
@@ -0,0 +1,390 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+require 'fileutils'
+require 'shellwords'
+require 'fog'
+
+# settings
+$autostart_clients = true
+
+# aws config
+$aws_akey   = ENV['AWS_AKEY']
+$aws_skey   = ENV['AWS_SKEY']
+$aws_region = "us-west-1"
+$aws_kpname = "akutz"
+$aws_secgrp = "pbs"
+$aws_ami    = "ami-de347abe" # Amazon Linux x64 PV EBS
+$aws_cnxn =   Fog::Compute.new(
+    :provider => :aws,
+    :region => $aws_region,
+    :aws_access_key_id => $aws_akey,
+    :aws_secret_access_key => $aws_skey)
+
+# ssh config
+$ssh_pkey   = "/Users/akutz/.ssh/ec2-akutz.pem"
```

Comment:
Do you want your PEM hardcoded in here? What if it was generic, e.g. `~/.ssh/ec2.pem` but overridable via ENV?